tests/playwright/sanity/bg-lazyload.test.js

```diff
@@ -0,0 +1,45 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../pages/wp-admin-page.js' );
+
+test( 'Background lazy load sanity test', async ( { page }, testInfo ) => {
+	const wpAdmin = new WpAdminPage( page, testInfo );
+	await wpAdmin.setExperiments( {
+		e_bg_lazyload: true,
+	} );
+
+	// Get the first lezyload background element.
+	const lazyloadSelector = '[data-e-bg-lazyload-src]:not([data-e-bg-lazyload-src=""])';
+
+	await page.goto( '/law-firm-about/' );
+	const element = await page.waitForSelector( lazyloadSelector );
+	expect( element ).toBeTruthy();
+
+	const beforeURL = await page.$eval( lazyloadSelector, ( el ) => {
+		const property = window.getComputedStyle( el ).getPropertyValue( 'background-image' );
+		return property.match( /url\((.*?)\)/ )[ 1 ].replace( /"/g, '' );
+	} );
+	expect( beforeURL ).toContain( '150x150.png' );
+
+	// eslint-disable-next-line no-shadow
+	await page.evaluate( ( lazyloadSelector ) => {
+		const lazyloadElement = document.querySelector( lazyloadSelector );
+		lazyloadElement.scrollIntoView();
+	}, lazyloadSelector );
+
+	await page.waitForTimeout( 1500 );
```

Comment:
Maybe use this?
https://playwright.dev/docs/api/class-page#page-wait-for-response

Reply:
Not relevant, the file content is updated.